client/landing/domains/transfer-away-confirmation/index.jsx

```diff
@@ -0,0 +1,343 @@
+				{
+					args: {
+						domain: domain,
+					},
+					components: {
+						strong: <strong />,
+						br: <br />,
+					},
+				}
+			),
+			messageAlignCenter: false,
+			actionTitle: 'Accept Transfer',
+			actionCallback: this.acceptTransfer,
+			alternateActionTitle: 'Cancel Transfer',
+			alternateActionCallback: this.cancelTransfer,
+			isLoading: false,
+		};
+	};
+
+	getEmailMismatchState = () => {
+		const { domain, translate } = this.props;
+
+		return {
+			message: translate(
+				'This email address is different from the one we have on record for {{strong}}%(domain)s{{/strong}}.',
+				{
+					args: {
+						domain: domain,
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			),
+		};
+	};
+
+	getWhoisErrorState = () => {
+		const { translate } = this.props;
+
+		return {
+			message: translate(
+				"Something went wrong.{{br/}}{{br/}}We couldn't get the transfer status for your domain.{{br/}}Please try again in a few hours.",
+				{
+					components: {
+						br: <br />,
+					},
+				}
+			),
+		};
+	};
+
+	getExpiredTokenErrorState = () => {
+		const { translate } = this.props;
+
+		return {
+			message: translate(
+				'This email has expired.{{br/}}{{br/}}If it has been over seven days since you have started your transfer and it has not yet completed, please {{a}}{{strong}}contact support{{/strong}}{{/a}}.',
+				{
+					components: {
+						a: <a href={ CALYPSO_CONTACT } />,
+						br: <br />,
+						strong: <strong />,
+					},
+				}
+			),
+			footer: null,
+		};
+	};
+
+	getNotPendingTransferErrorState = () => {
+		const { translate } = this.props;
+
+		return {
+			title: translate( 'Domain not awaiting transfer.' ),
+			message: translate(
+				'This domain is no longer pending transfer.{{br/}}{{br/}}If your transfer has completed successfully or you previously cancelled your domain transfer, there is nothing else you need to do.',
+				{
+					components: {
+						br: <br />,
+					},
+				}
+			),
+		};
+	};
+
+	getDefaultErrorState = () => {
+		const { translate } = this.props;
+		const defaultErrorFooter = translate(
+			"If you're having trouble managing your domain transfer, please {{a}}{{strong}}contact support{{/strong}}{{/a}}.",
+			{
+				components: {
+					a: <a href={ CALYPSO_CONTACT } />,
+					strong: <strong />,
+				},
+			}
+		);
+
+		return {
+			title: translate( 'Uh oh!' ),
+			message: translate( 'Hmm. Something went wrong.' ),
+			messageAlignCenter: true,
+			actionTitle: null,
+			actionCallback: null,
+			alternateActionTitle: null,
+			alternateActionCallback: null,
+			footer: defaultErrorFooter,
+			isLoading: false,
+			isProcessingRequest: false,
+		};
+	};
+
+	setErrorState = error => {
+		let errorState;
+
+		switch ( error.error ) {
+			case 'email_mismatch':
+				errorState = this.getEmailMismatchState();
+				break;
+
+			case 'token_expired':
+				errorState = this.getExpiredTokenErrorState();
+				break;
+
+			case 'whois_status_error':
```

Comment:
This seems to be `whois_error` on the backend?

Reply:
Yep. Thanks!